sdccc/src/main/java/com/draeger/medical/sdccc/TestSuite.java

```diff
@@ -165,13 +165,11 @@ public long runTestSuite() {
         final SummaryGeneratingListener directSummary = new SummaryGeneratingListener();
         directTestLauncher.registerTestExecutionListeners(directSummary);
 
-        final Boolean isConsumerEnabled = setupDevice();
```

Comment:
Why did you move this into phase 1?? This is still the setup before running phase 1, i.e. generating messages.